grants/milestone-deliverables-guidelines.md

```diff
@@ -1,28 +1,82 @@
 # Milestone Deliverables Guidelines
 
-We kindly ask you to consider the following guidelines when you submit a milestone for evaluation. Please submit your milestones by email to grants@web3.foundation.
+We kindly ask you to consider the following guidelines when you submit a milestone for evaluation. 
+Please submit your milestones by email to grants@web3.foundation.
 
 The submission should contain the following information: 
  * **Name of the grant project**
  * **Link to the open-source code/delivery** 
- * **Basic [testing guide](#testing-guide)** 
+ * **[License](#license)**
+ * **[Documentation](#documentation)**
+ * **[Testing Guide](#testing-guide)**
+ * **[End-user Tutorial](#end-user-tutorial)**
  * **A list of the [milestone deliverables](#milestone-deliverables)**
  * **Any [additional information](#additional-information)**
 
+## License
+
+In order to successfully receive grant funding for your application it is necessary for the project to have open source code. 
+We prefer Apache 2.0, but the GNU GPL v3 license is also acceptable. 
```

Comment:
I would change it to "but the MIT or the Unlicense are also acceptable." We only accept GNU GPL v3 because of substrate, ideally in other cases it actually shouldn't be used.